src/content/docs/iast/overview.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Introduction to New Relic IAST
+tags:
+    - New Relic IAST
+    - IAST
+    - Exploitable vulnerabilities
+metaDescription: Learn what New Relic IAST is and how you can use it.
+redirects:
+    - /docs/iast/use-iast
+---
+
+import iastSummary from 'images/iast_screenshot-crop_summary.webp'
+
+
+<Callout title="PREVIEW"> 
+    This feature is currently in preview. To celebrate it, we're offering you a 3-month free trial! The IAST free trial starts the moment you accept the pre-release software terms. 
+
+    IAST must only be used in a pre-production environment, ideally in a dedicated security environment.
+    
+    Use of the IAST agent may generate additional billable APM data ingest.
+</Callout>
+
+Nowadays, the security of your software applications is essential to protect sensitive data, maintain the trust of your users, and comply with regulatory requirements. New Relic Interactive Application Security Testing (IAST) can help you to prevent cyberattacks and breaches on your applications.
+
+IAST enables you to focus on real security risks, remediate issues faster across the application lifecycle, and ship secure code with speed and confidence. With New Relic IAST, you can see all protected and unprotected applications, which eliminates blind spots and hidden threats. Comprehensive dashboards provide complete visibility into high-priority vulnerabilities, from applications to their components and APIs. 
```

Comment:
These paragraphs feel very marketing-y. I wonder if they're redundant to the next paragraph+bullets?

Reply:
Yes, I've changed it.